client.go

```diff
@@ -6,11 +6,14 @@ package kmip
 
 import (
 	"crypto/tls"
+	"fmt"
 	"time"
 
 	"github.com/pkg/errors"
 )
 
+var ErrResponseType = fmt.Errorf("unexpected response type")
```

Comment:
Sorry, last nitpick. 
```suggestion
	"errors"
	"time"

	"github.com/pkg/errors"
)

var ErrResponseType = errors.New("unexpected response type")
```